### ISSUE TITLE
funasr-runtime-sdk-gpu-0.2.0 windows 11 podman 无法启动

### ISSUE THREAD

**Issue by lonelyxmas** [modelscope/FunASR#2433]
Notice: In order to resolve issues more efficiently, please raise issue following the template.
（注意：为了更加高效率解决您遇到的问题，请按照模板提问，补充细节）

## ❓ Questions and Help

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.987 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.985, %5760, %5761, %5762), scope: __module.model.encoders/__module.model.encoders.48/__module.model.encoders.48.self_attn/__module.model.encoders.48.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

Killed
I20250315 02:56:46.004922   682 funasr-wss-server.cpp:308] Failed to download model from modelscope. If you set local asr model path, you can ignore the errors.
E20250315 02:56:46.012068   682 funasr-wss-server.cpp:312] /workspace/models/damo/speech_paraformer-large-vad-punc_asr_nat-zh-cn-16k-common-vocab8404-pytorch/model_blade.torchscript do not exists.

### Before asking:
1. search the issues.
2. search the docs.

#### What's your environment?

windows11 
podman desktop 
funasr-runtime-sdk-gpu-0.2.0
podman run --device nvidia.com/gpu=all -p 10098:10095 -idt --privileged=true `
--name funasr `
-v D:\cloud\funasr\models:/workspace/models `
registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.2.0
